fvm/state/state.go

```diff
@@ -320,11 +318,8 @@ func IsFVMStateKey(owner, controller, key string) bool {
 func PrintableKey(key string) string {
 	// slab
 	if key[0] == '$' && len(key) == 9 {
-		i := uint64(binary.LittleEndian.Uint64([]byte(key)))
+		i := uint64(binary.BigEndian.Uint64([]byte(key[1:])))
 		return fmt.Sprintf("$%d", i)
 	}
-	if !utf8.ValidString(key) {
-		return strings.ToValidUTF8(key, "?")
-	}
-	return key
+	return fmt.Sprintf("#%x", []byte(key))
```

Comment:
What is the reasoning behind a `#` prefix? could we use no prefix or a `0x` prefix?

Reply:
No real reason, just seems like a nice counterpart for `$` and using `0x` would make it look like a number